README.md

```diff
@@ -113,13 +113,13 @@ Here are the main commands you will want to run. For a more exhaustive explanati
 
 Aside from the command line interface, this crate can also be used as a library. This means that you could integrate it in an existing project, like a web-app for example. Since the command line interface is just a wrapper around the library functionality, when you use this crate as a library you have full access to all the functionality of the command line interface with an easy to use API and more!
 
-See the [Documentation](http://azerupi.github.io/mdBook/lib/lib.html) and the [API docs](http://azerupi.github.io/mdBook/mdbook/index.html) for more information.
+See the [Documentation](http://rust-lang-nursery.github.io/mdBook/lib/lib.html) and the [API docs](https://docs.rs/mdbook/0.0.26/mdbook/) for more information.
```

Comment:
the api docs are still available under http://rust-lang-nursery.github.io/mdBook/mdbook/index.html

if you really like to point to docs.rs then I'd suggest pointing to wildcard version 

https://docs.rs/mdbook/*/mdbook
which will always point to the newest version

Reply:
I'd prefer to point at docs.rs here, that way it's showing the docs for the latest release and not whatever happens to be in `master`. Later on if necessary, we can always add something to the README which gives links to both.

```md
Documentation:

- [master](http://rust-lang-nursery.github.io/mdBook/lib/lib.html)
- [Latest Release](https://docs.rs/mdbook/*/mdbook/)
```

Also, what about instead of using the word "Documentation" for the first link we say "User Guide"? That helps to differentiate between the API docs (used by people hacking on mdbook) and the user guide.

Reply:
Thanks @budziq and @Michael-F-Bryan for the review.  So just checking:
[User Guide](http://rust-lang-nursery.github.io/mdBook/)
[API docs](https://docs.rs/mdbook/*/mdbook/)

Is that right?